src/main/webapp/Search.jsp

```diff
@@ -0,0 +1,81 @@
+<%@ page errorPage="Error.jsp" %>
+<%@ page import="java.sql.*" %>
+<%@ page import="main.java.com.registration.*" %>
+<%@ page import="main.java.com.registration.dbconnection.*" %>
+<% Class.forName("com.mysql.jdbc.Driver"); %>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>HomePage</title>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto|Varela+Round|Open+Sans">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<!-- Main css -->
+    <link rel="stylesheet" href="css/HomePage.css">
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
+</head>
+
+<body>
+<%@ include file="Navbar.jsp" %>
+    <%
+           String name = request.getParameter("search");
+           Connection connection = DriverManager.getConnection(
+             "jdbc:mysql://localhost:3306/javaproject", "root", "123456");
```

Comment:
Can you do the modification in your code in a way so we don't need to write JDBC code in the JSP file? The JSP file should only be used to display the information.